src/content-distribution/distribute-panel/index.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * WordPress dependencies.
+ */
+import { __ } from '@wordpress/i18n';
+import { PluginSidebar } from '@wordpress/editor';
+import { Flex, Panel, PanelBody } from '@wordpress/components';
+import { globe } from '@wordpress/icons';
+
+/**
+ * Internal dependencies.
+ */
+import './style.scss';
+
+const DistributePanel = ({ header, body, footer, buttons }) => {
```

Comment:
Since we have the `Distribute()` component/plugin, maybe we can call this `ContentDistributionPanel`?

Reply:
Yeah, that makes sense. Unless you think renaming the `Distribute()` component to `OutgoingPost` to match the backend code and this one?

Reply:
> Unless you think renaming the Distribute() component to OutgoingPost to match the backend code and this one?

Hmm, I might. Either way, I think `ContentDistributionPanel` is more suitable.